package.json

```diff
@@ -22,7 +22,7 @@
   "author": "Yiding Wang",
   "license": "MIT",
   "dependencies": {
-    "cdnup": "^2.5.0",
+    "cdnup": "^4.0.0",
```

Comment:
that's quite the bump, no side effect here?

Reply:
@msluther its to get your change and yea it will be fine.

Reply:
not for our use case.